src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1480,6 +1480,23 @@ def _get_child_object_display_names(self) -> list[str]:
             )
         return child_object_display_names
 
+    def _get_id_name(self):
+        child_object_id_name_list = []
+        obj_type = self.path[-1][0]
+        for name in self._get_child_object_names():
+            child_object_id_name_list.append(obj_type + ":" + name)
+        return child_object_id_name_list
```

Comment:
@prmukherj So this is returning a list like `["TaskObject:Import Geometry",  "TaskObject:Add Local Sizing"]`?